Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/QuantitySelector.kt

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("DEPRECATION") // FIXME move to new ConstraintLayout dep once released
```

Comment:
Can we suppress just the function which uses ConstraintLayout? `file` suppressions tends to hide things which we can fix.

Reply:
As discussed, there's also a bunch of deprecated imports so leaving as file suppress. Hopefully we can remove this shortly when `constraintlayout-compose` `alpha02` comes out.